packages/core/src/module-resolver.ts

```diff
@@ -1371,9 +1371,14 @@ export class Resolver {
     // really be understood as a request for a module in the containing engine
     if (owningPackage.isV2Addon()) {
       let sections = [owningPackage.meta['app-js'], owningPackage.meta['fastboot-js']];
+      let fromPackageRelativePath = explicitRelative(owningPackage.root, fromFile);
```

Comment:
I don't know if it is common for the sections to both be `undefined`, if so this change would hit `explicitRelative` more often which I could guard against